badge-maker/lib/badge-cli.js

```diff
@@ -18,7 +17,7 @@ if (process.argv.length < 4) {
   console.log('    #xxxxxx (six hex digits)')
   console.log('    color (CSS color)')
   console.log('  output:')
-  console.log('    svg, png, jpg, or gif')
+  console.log('    svg or json')
```

Comment:
Is there a reason the CLI ought to support JSON? I'm not sure I see the value of that.

Reply:
This isn't actually new - it worked before it just wasn't documented in the console help.

Getting rid of the raster support allowed us to drop some additional dependencies but maintaining this is zero effort/overhead so I'm inclined to just keep it around. In fact, we'd have to add an extra validation rule to disallow it in this case.

Reply:
It doesn't really make sense as a feature, though. If it's not documented, we could remove it later without calling it a breaking change. This could be considered in a follow-on PR, post rc1.

Reply:
> This could be considered in a follow-on PR, post rc1.

Meh. Its taken the thick end of a year to get as far as opening this PR. If we need another evening or two to get this detail right I think we can all cope.

> If it's not documented, we could remove it later without calling it a breaking change

This is an interesting point. I hadn't focussed that we've never previously documented that this exists at all. I'm not deeply concerned about a breaking change given the laundry list of other breaking changes in this release. That fact we've never documented it does substantially reduce the chance that anyone is using it though :slightly_smiling_face: 

Thinking about it a bit further, if we drop JSON from the CLI, we should just drop the format argument completely. If the only possible valid argument you can pass is `.svg` and its optional, `badge build passed :green .svg` serves no purpose at all.

I'll have a look at this later..